PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+- [ ] Run tests
+- [ ] Update the [`Changelog.md`][1] with a summary of your changes
```

Comment:
Nit: Might as well reference it as `CHANGELOG.md` to prevent any file case confusion?